box-share-sdk/src/main/java/com/box/androidsdk/share/api/BoxShareController.java

```diff
@@ -40,13 +40,15 @@
     private BoxApiCollaboration mCollabApi;
     private BoxApiInvitee mInviteeApi;
     private BoxApiFeatures mFeaturesApi;
+    private String mUserId;
 
-    public BoxShareController(BoxApiFile fileApi, BoxApiFolder folderApi, BoxApiBookmark bookmarkApi, BoxApiCollaboration collaborationApi, BoxApiInvitee inviteeApi) {
+    public BoxShareController(BoxApiFile fileApi, BoxApiFolder folderApi, BoxApiBookmark bookmarkApi, BoxApiCollaboration collaborationApi, BoxApiInvitee inviteeApi, String userId) {
         mFileApi = fileApi;
```

Comment:
I'd prefer not including userId as part of the constructor. If necessary a getter for the session itself is preferable. For our purposes we have BoxSession as a singleton and so the user Id can change through that, however with this there is a possibility we persist the share controller with an old user id.